### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by ovarte**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
image: /assets/2020/your-blog-title/teaser.jpg
---

YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST

```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/assets/2020` directory.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.js.org#contribute-guideline)

### CODE REVIEW

File: _posts/2020-08-26-statistical-robot.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: "群订餐助手"
+author: ovarte
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-padplus
+  - order
+image:
```
Comment (lijiarui) [inline _posts/2020-08-26-statistical-robot.md:9]: Please add a image for this blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by lijiarui** [comment]
ping @ovarte

**Comment by wj-Mcat** [comment]
ping @ovarte . If you have not fix this pr at next three days,  we will close it. Thanks a lot.

**Comment by huan** [comment]
This is a Python Wechaty example! @wechaty/python

**Comment by huan** [comment]
Close this PR due to no activities for more than 3 months.

**Pull request closed without merging**